repository-meta-analyzer/src/test/java/org/acme/RepositoryMetaAnalyzerTopologyTest.java

```diff
@@ -56,14 +57,29 @@ public class RepositoryMetaAnalyzerTopologyTest {
     private TestInputTopic<UUID, Component> inputTopic;
     private TestOutputTopic<UUID, MetaModel> outputTopic;
 
+    @InjectMock
     private ComposerMetaAnalyzer composerMetaAnalyzerMock;
+    @InjectMock
     private GemMetaAnalyzer gemMetaAnalyzerMock;
+    @InjectMock
     private GoModulesMetaAnalyzer goModulesMetaAnalyzerMock;
+
+    @InjectMock
     private HexMetaAnalyzer hexMetaAnalyzerMock;
+
+    @InjectMock
     private MavenMetaAnalyzer mavenMetaAnalyzerMock;
+
+    @InjectMock
     private NpmMetaAnalyzer npmMetaAnalyzerMock;
+
+    @InjectMock
     private NugetMetaAnalyzer nugetMetaAnalyzerMock;
+
+    @InjectMock
     private PypiMetaAnalyzer pypiMetaAnalyzerMock;
+
+    @InjectMock
```

Comment:
As you're using `@InjectMock` now, you can remove the mocking logic in `beforeEach`.